tests/swarm/test_bandwidth_governor.py

```
@@ -0,0 +1,460 @@
+        # Send critical messages (should go through)
+        for i in range(5):
+            result = governor.acquire_tokens(
+                peer,
+                500,  # 500B each
+                MessagePriority.CRITICAL
+            )
+            # Most should succeed due to low global utilization
+    
+    def test_dos_prevention_single_talker(self):
+        """Test DoS prevention when one agent overtalks."""
+        config, agent_id = create_config(num_peers=5)
```

Ruff (0.14.10)

Unpacked variable `agent_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)